src/main/resources/avro/reads.avdl

```diff
@@ -202,6 +207,12 @@ record ReadAlignment {
   The read ordinal in the fragment, 0-based and less than numberReads. This
   field replaces SAM flag 0x40 and 0x80 and is intended to more cleanly
   represent multiple reads per fragment.
+  A readNumber of 0 is equivalent to SAM flag 0x40 being set and 0x80 being
+  unset.
+  A readNumber of 1 is equivalent to SAM flag 0x80 being set and 0x40 being
+  unset.
+  A readNumber other than 0 or 1 means both SAM flags 0x40 and 0x80 are set.
```

Comment:
I think that the current comment is clear and OK.  This is saying very clearly that each GA4GH API value of 
readNumber is equivalent to a particular combination of SAM flag states - there is no single equivalence to
any one tag.  It doesn’t matter whether flags are set or unset in the API - they are in SAM, and it is
their SAM state that is being referred to.
 
> On 16 Dec 2015, at 02:47, CH Albach <notifications@github.com> wrote:
> 
> In src/main/resources/avro/reads.avdl <https://github.com/ga4gh/schemas/pull/509#discussion_r47731145>:
> 
> > @@ -202,6 +207,12 @@ record ReadAlignment {
> >    The read ordinal in the fragment, 0-based and less than numberReads. This
> >    field replaces SAM flag 0x40 and 0x80 and is intended to more cleanly
> >    represent multiple reads per fragment.
> > +  A readNumber of 0 is equivalent to SAM flag 0x40 being set and 0x80 being
> > +  unset.
> > +  A readNumber of 1 is equivalent to SAM flag 0x80 being set and 0x40 being
> > +  unset.
> > +  A readNumber other than 0 or 1 means both SAM flags 0x40 and 0x80 are set.
> The wording throughout this PR seems off to me. SAM flags are not 'set' in the GA4GH APIs. Some property of the GA4GH API may be equivalent to a SAM flag, for instance our comment for properPlacement: (equivalent to SAM flag 0x2).
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/ga4gh/schemas/pull/509/files#r47731145>.
> 




-- 
 The Wellcome Trust Sanger Institute is operated by Genome Research 
 Limited, a charity registered in England with number 1021457 and a 
 company registered in England with number 2742969, whose registered 
 office is 215 Euston Road, London, NW1 2BE.